certbot-postfix/local-oldest-requirements.txt

```diff
@@ -0,0 +1,2 @@
+-e acme[dev]
```

Comment:
Now that acme 0.25.0 is out, this can be `acme==0.25.0`. I should have been more descriptive in my comment as this change just landed for the other files in master.